src/main/resources/data/spectrum/modonomicon/books/guidebook/entries/decoration/banner_patterns.json

```diff
@@ -30,7 +30,7 @@
       "text": "Resources and tools you come across can be used directly in the Loom as patterns."
     },
     {
-      "type": "modonomicon:crafting",
+      "type": "modonomicon:crafting_recipe_recipe_recipe_recipe",
```

Comment:
I think this might have replaced a few too many times

Reply:
huh weird